pylint/test/functional/mutable_sequence_modified_in_loop.py

```diff
@@ -0,0 +1,75 @@
+
+
+""" The mutable sequence was modified inside the loop """
+
+# pylint:disable=print-statement, missing-docstring, redefined-outer-name, invalid-name, using-constant-test, no-member
+
+from __future__ import print_function
+
+
+class MyList(object):
+
+    def __init__(self):
+        self.my_list = range(1, 6)
+
+    def __iter__(self):
+        return iter(self.my_list)
+
+    def remove(self, item):
+        pass
+
+    def delete(self, item):
+        pass
+
+    def pop(self, item):
+        pass
+
+
+my_list = MyList()
+for item in my_list:
+    print(item)
+    my_list.remove(item)
+
+my_list = MyList()
+for item in my_list:
+    print(item)
+    my_list.delete(item)
+
+my_list = MyList()
+for item in my_list:
+    print(item)
+    my_list.pop(item)
+
+
+items = range(1, 6)
+for item in items:
+    print(item)
+    items.remove(item)  # [mutable-sequence-modified-in-loop]
```

Comment:
I don't understand
specifically what should activated only in python 2 is `items.remove(item)` ?
Because ?